### DISCUSSION TITLE
Mac için GoodbyeDPI ve PCden Wifi sharing sorunu

### DISCUSSION THREAD

**Discussion by Technocrevv** [cagritaskn/GoodbyeDPI-Turkey#16] [Ideas]
Acaba bu programın mac versiyonu için de çalışmalar olacak mı? Ben de sorunsuz çalışıyor. Fakat Pc üzerinden sharing yaptığımda çalışmıyor. Bununla alakalı bir bilginiz var mıdır? Teşekkürler.

**Comment by cagritaskn** [comment]
Program şuan için hotspot/AP sharing ile uyumlu çalışmıyor. Bu özelliğin kullanılması için güncelleme yapılırsa ben de güncelleme yapacağım.

**Reply by Technocrevv** [reply]
Çok teşekkürler. Beklemekteyim.
